registry.json

```diff
@@ -117,6 +117,21 @@
       "telegram":"CardanoPoolU",
       "operatorName":"RALPH",
       "github":""
+   },
+      {
+      "ticker":"LGLP",
+      "poolId":"6936fe0511eada38cbc229ce57dda32b8db2a27e237689b46afa4228",
+      "social":{
+         "facebook":"",
+         "twitter":"LglpStudios",
+         "website":"https://twitter.com/LglpStudios",
```

Comment:
Please can you add a personal website or leave "website" value blank?

Reply:
I'll remove the website link - my main exposure is through twitter at this time.